config/routes.rb

```diff
@@ -2,6 +2,9 @@
   devise_for :users, controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }
   root 'home#index'
   get 'home', to: 'home#index', as: 'home'
+  get 'all_users', to: 'users#index'
```

Comment:
I think you can simplify this: remove line 5 and remove except from line 7.

Reply:
thank you for this notice! done ✅